### ISSUE TITLE
Local Uploads 404'ing

### ISSUE THREAD

**Issue by stevesbrain** [turtl/api#11]
Hi there,

I've set up my config.lisp as so:

```lisp
(defvar *local-upload* "/uploads"
         )
     (defvar *local-upload-url* "http://url.com/uploads"
         )
```

I can see the file uploads successfully:

```bash

$ ls -ls uploads
total 88
88 -rw-r--r-- 1 user user 89063 Jul  4 08:17 0155b48dcf397f5f1d8917ae02eeb170134c2dfa1e0e2423350b385bbc369f52f1f1964de0640124
```

However, when trying to load the file, it 404's:
```bash
  <INFO> [1467615352] wookie - GET /uploads/files/0155b48dcf397f5f1d8917ae02eeb170134c2dfa1e0e2423350b385bbc369f52f1f1964de0640124 404 15
```

This makes me believe I've got the formatting wrong for local-upload-url - however, what should this look like if local-upload is defined then? Thanks in advance!

**Comment by stevesbrain** [comment]
Okay, so I just tried taking the "/uploads" away from the URL - that worked. I was getting confused when the documentation mentioned it had to be defined if a path was specified - my bad! As an aside, when I fixed this, and logged back in afterward, then tried clicking on the image, it still failed. It only pulled down when I deleted all local data and rescyn'd. Would this have "cached" a copy pointing at the wrong URL - or is this expected behaviour any time an image does not load correctly initially? Thanks!
